pkg/app/api/service/webservice/service.proto

```diff
@@ -179,6 +180,20 @@ message AddApplicationResponse {
     string application_id = 1 [(validate.rules).string.min_len = 1];
 }
 
+message UpdateApplicationRequest {
+    string application_id = 1 [(validate.rules).string.min_len = 1];
+    string name = 2 [(validate.rules).string.min_len = 1];
+    string env_id = 3 [(validate.rules).string.min_len = 1];
+    string piped_id = 4 [(validate.rules).string.min_len = 1];
+    model.ApplicationGitPath git_path = 5 [(validate.rules).message.required = true];
+    model.ApplicationKind kind = 6 [(validate.rules).enum.defined_only = true];
+    string cloud_provider = 7 [(validate.rules).string.min_len = 1];
+}
+
+message UpdateApplicationResponse {
+    bool completed = 1;
```

Comment:
Is this necessary? No error already means the operation was successfully completed.

Reply:
I don't have strong opinion about this 😅 let's remove the field 🆗